packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.scss

```diff
@@ -90,7 +90,6 @@ a.ms-ContextualMenu-link {
 }
 
 .ms-ContextualMenu-icon {
```

Comment:
@dzearing 
David, when I run ```npmx change```, I'm getting below (ran from the root of the repo):

    NPMX Multi-Package Build Tool 1.2.1

    No change file is needed.

Is this expected. Am I doing something wrong?